lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -13,6 +13,9 @@
 end
 
 require "zlib"
+require "open3"
```

Comment:
(nit) we could move this to just before the first use of `Open3` to reduce boot time by a tiny bit.